workbench/runtime.py

```diff
@@ -261,8 +266,46 @@ def __init__(self, user_id=None):
         self.id_generator = ID_MANAGER
         self.user_id = user_id
 
+    def get_user_role(self):
+        """Provide a dummy user role."""
+        return 'Student'
+
+    @property
+    def descriptor_runtime(self):
+        """Provide a dummy course."""
+        course = Mock(
+            lti_passports=['test:test:secret'],
+            display_name_with_default='Test Course',
+            display_org_with_default='edX',
+        )
+
+        return Mock(modulestore=Mock(
+            get_course=Mock(return_value=course)
+        ))
+
+    def get_real_user(self, _):
+        """Return a dummy user."""
```

Comment:
The method name and its docstring seem to disagree...

Reply:
Yeah, I've added a better docstring.